index.html

```diff
@@ -0,0 +1,162 @@
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+<body>
+    
+
+    <div class="main">
+     
+      <header id="header">
+        <div id="header-img">
+          <img src=https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png width="300px">
+          </div>
+          <nav id="nav-bar">
+     <ul>
+         <li><a href="#Features" class="nav-link">Features</a></li>
+         <li><a href="#How It Works" class="nav-link">How it Works</a></li>
+         <li><a href="#Pricing" class="nav-link">Pricing</a></li>
+     </ul>
+  </nav>
+     </header>
+    
+  
+     <div class="container-subscriber">
+       <form action="https://www.freecodecamp.com/email-submit">
+  
+              <h1>Handcrafted, home-made masterpieces</h1>
+              <input type="email" class="email" placeholder="Enter your email address" required><br>
+              <input type="submit" id="submit" value="GET STARTED">
+  
+       </form>
+  
+    </div>
+  
+    <section id="Features" class="container">
+    
+     <div id="containerBox">
+     <div class="container-icon">
+         <i class="fa fa-3x fa-fire"></i>
+         <img src="https://www.iconsdb.com/icons/preview/orange/fire-2-xxl.png" alt="fire">
+   
+     </div>
+     <div class="container-text">
+         <h1>Premium Materials</h1>
+         <p>Our trombones use the shiniest brass which is sourced locally.
+              This will increase the longevity of your purchase.</p>
+         
+     </div>
+   
+      </div>
+  
+          <div id="containerBox">
+         <div class="container-icon">
+             <i class="fa fa-3x fa-truck"></i>
+             <img src="https://www.iconsdb.com/icons/preview/orange/truck-2-xxl.png" alt="truck">
+          </div>
+         <div class="container-text">
+             <h1>Fast Shipping</h1>
+             <p>We make sure you recieve your trombone as soon as we have finished making it.
+                  We also provide free returns if you are not satisfied.</p>
+             
+         </div>
+       
+          </div>
+          <div id="containerBox">
+              <div class="container-icon">
+            <i class="fa fa-3x fa-battery-full"></i>
+            <img src="https://www.iconsdb.com/icons/preview/orange/full-battery-xxl.png" alt="battery">
+              </div>
+              <div class="container-text">
+                  <h1>Quality Assurance</h1>
+                  <p>For every purchase you make, we will ensure there ar
+                      e no damages or faults and we will check and test the pitch of your instrument.</p>
+            
+              </div>
+            
+               </div>
+  
+    </section>
+  
+  
+    <section id="HowItWorks" class="video">
+  
+      <iframe width="560" height="315" src="https://www.youtube.com/embed/y8Yv4pnO7qc?start=44" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; 
```

Comment:
Good comment as this can be a problem, especially with linters but for better communication, you could formulate it like this:
**Thank you for your hard work on this project. I noticed that some of the lines of code are quite long, exceeding 100 characters. While there is no hard and fast rule for how long a line of code should be, it's generally a good idea to keep them short and easy to read.**

I hope that helps, let me know if you have any questions 🙂